src/script/lib/constants.ts

```diff
@@ -2,6 +2,12 @@ export const defaultSettingValues = {
   ALLOW_SCREENSHOT: true,
   SAVE_IMAGE: true,
   ALWAYS_PRESENT: true,
+  SEND_UNSAVEABLE: false,
+  SNAP_AS_MOBILE: true,
```

Comment:
I think this should be default false, best way to approach the default settings is setting it what snapchat normally does. Unless its a quality of life change like allow screenshots and etc